src/components/Atoms/Input/Input.js

```diff
@@ -97,6 +101,7 @@ Input.propTypes = {
   label: PropTypes.string.isRequired,
   placeholder: PropTypes.string.isRequired,
   errorMsg: PropTypes.string.isRequired,
+  showLabel: PropTypes.string.isRequired,
```

Comment:
Can you use `boolean`?

Reply:
Yes sure, it's possible but because styled components convert boolean to string I thought to use yes or no work so that we don't have to do extra work.